src/mod_websockets.erl

```diff
@@ -38,16 +39,18 @@
 
 -record(websocket, {
           pid :: pid(),
-          peername :: {inet:ip_address(), inet:port_number()}
+          peername :: mongoose_transport:peer(),
+          peercert :: undefined | binary()
```

Comment:
peername -> peer_name ??
peercert -> peer_cert ??

Reply:
Again, consistency with `ssl:peername` and `ssl:peercert`. It's up to you if you still want it changed.

Reply:
So is ok :)